icicle-compiler/src/Icicle/Sea/FromAvalanche/Base.hs

```diff
@@ -91,7 +91,7 @@ mangleToSeaName = SeaName . Text.pack . concatMap mangle . show . pretty
      | c >= 'A' && c <= 'Z'
      = [c]
      | c == '_'
-     = "__"
+     = "_"
```

Comment:
I'd prefer the second: it's more like escaping a string than finding a normal form.